addons/l10n_in/data/template/account.tax-in.csv

```diff
@@ -766,3 +766,111 @@
 "","","","","","","","","","","","","","","","","","base","refund","",""
 "","","","","","","","","","","","","","","","","","tax","refund","p10054",""
 "","","","","","","","","","","","","","","","","-100","tax","refund","p10054",""
+"cess_non_itc_purchase_5","5% CESS P Non ITC 17(5)","CESS Non ITC u/s 17(5)","CESS 5% Non ITC u/s 17(5)","none","percent","5.0","","","cess_group","","","","","","","","base","invoice","","l10n_in.tax_tag_non_itc_base_cess"

```

Comment:
The taxes should probably be inactive as discussed on the task
```suggestion
"cess_non_itc_purchase_5","5% CESS P Non ITC 17(5)","CESS Non ITC u/s 17(5)","CESS 5% Non ITC u/s 17(5)","none","percent","5.0","","False","cess_group","","","","","","","","base","invoice","","l10n_in.tax_tag_non_itc_base_cess"

```

Reply:
The discussion between SHID and CHKL is still ongoing, as is evident in the task. Once they reach a decision, I will make the necessary changes accordingly. Thank you.